mimic_2_data_analysis.ipynb

```diff
@@ -0,0 +1,3433 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    data=data.dropna()
</pre><p>Why is this required? Why should have any NAs now?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/ehrapy-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='132'/>

Reply:
<p>For CoxPHFitter.Fit(), it cannot deal with NaNs. </p><p>In this notebook, we don't have any preprocessing part, i.e. no imputation. data=data.dropna() will drop one row which contains NaN. </p>